### PR TITLE
harcode default WTS url

### PR DESCRIPTION
**Pull request by jawadqur**
Fix so gen3-fuse still works in local cluster, as well as external cluster

### CODE REVIEW

File: sidecarDockerrun.sh [hunk 1]
```diff
@@ -142,8 +146,14 @@ mount_manifest() {
     # gen3-fuse mounts the files in /data/<hostname> dir
     if [ ! -d $IDP_DATA_PATH/$MOUNT_NAME ]; then
         echo "Mounting manifest at $IDP_DATA_PATH/$MOUNT_NAME"
-        if (! gen3-fuse -config=/fuse-config.yaml -manifest=$PATH_TO_MANIFEST -mount-point=$IDP_DATA_PATH/$MOUNT_NAME -hostname=$BASE_URL -wtsURL=$WTS_URL -wtsIDP=$IDP -access-token=$ACCESS_TOKEN >/proc/1/fd/1 2>/proc/1/fd/2); then
-            echo "gen3-fuse failed to mount."
+        if [ -z $ACCESS_TOKEN ]; then
```
Comment (ZakirG) [inline sidecarDockerrun.sh:149]: Should this `$ACCESS_TOKEN` variable be changed to `TOKEN_JSON[$IDP]`?

File: sidecarDockerrun.sh [hunk 2]
```diff
@@ -142,8 +146,14 @@ mount_manifest() {
...
+            if (! gen3-fuse -config=/fuse-config.yaml -manifest=$PATH_TO_MANIFEST -mount-point=$IDP_DATA_PATH/$MOUNT_NAME -hostname=$BASE_URL -wtsURL=$WTS_URL -wtsIDP=$IDP >/proc/1/fd/1 2>/proc/1/fd/2); then
+                echo "gen3-fuse failed to mount."
+            fi
+        else
+            if (! gen3-fuse -config=/fuse-config.yaml -manifest=$PATH_TO_MANIFEST -mount-point=$IDP_DATA_PATH/$MOUNT_NAME -hostname=$BASE_URL -wtsURL=$WTS_URL -wtsIDP=$IDP -access-token=$ACCESS_TOKEN >/proc/1/fd/1 2>/proc/1/fd/2); then
+                echo "gen3-fuse failed to mount."
```
Comment (ZakirG) [inline sidecarDockerrun.sh:155]: Can you add a clarifying comment here pointing out the new access-token argument in the second fuse mount call, and why it's needed?

**Review by ZakirG — APPROVED** [review]

File: sidecarDockerrun.sh (hunk 2 above)
Comment (jawadqur) [reply sidecarDockerrun.sh:156]: Added

File: sidecarDockerrun.sh (hunk 1 above)
Comment (jawadqur) [reply sidecarDockerrun.sh:149]: Not really, what we really are checking for whether or not it's a standard (local) workspace pod, or an external pod. Only external pods get access token mounted.

**Pull request merged**